clusterloader2/pkg/prometheus/manifests/00monitoring-priorityclass.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: scheduling.k8s.io/v1
+kind: PriorityClass
+metadata:
+  name: monitoring
+value: 1000000
+globalDefault: false
+description: "This priority class should be used for monitoring pods only."
```

Comment:
While in principle I agree with this approach, I would like to understand the consequences for Kubemark better.
- What will happen if we preempt a bunch of hollow-nodes to make place for prometheus?
- Don't we need to add some additional logic to ensure that they will really be up when we will be starting tests?
- Don't we need to add some logic to ensure that the preempted ones will be cleaned-up (and nothing will get scheduled later on them?)

There are couple questions that I would like to see clear answers for.

Reply:
Discussed in person.

Prometheus having a higher priority will evict some hollow-nodes, which can

1. New hollow-nodes won't be registered before an actual test starts. This is not a high risk. 
2. Old hollow-nodes won't be remove and be used for scheduling for some time (it'd be worth to find out what "some time" is in this case), which can cause unwanted behavior in tests (e.g. failing pod startup latency). 

We decided not to proceed with this PR because of 2. and instead add another, tainted node, on which hollow-nodes won't be scheduled.